data/json/items/tool_armor.json

```diff
@@ -2211,6 +2211,51 @@
     },
     {
         "type" : "TOOL_ARMOR",
+        "id" : "bagpipes",
+        "name" : "bagpipes",
+        "name_plural": "bagpipes",
+        "category" : "tools",
+        "weight" : 2300,
+        "color" : "green",
+        "covers" : ["TORSO"],
```

Comment:
Checked Wiki and didn't see any art of pipers both playing and engaging in combat at the same time.  :-/